qmlui/qml/UISettingsEditor.qml

```diff
@@ -30,13 +30,17 @@ Rectangle
     anchors.fill: parent
     color: "transparent"
 
-    property real origItemHeight: UISettings.listItemHeight
-    property real origIconMedium: UISettings.iconSizeMedium
+    property real origItemHeight: { origItemHeight = UISettings.listItemHeight }
```

Comment:
Are you sure this syntax makes sense?

Reply:
This syntax is meant to prevent a binding from being created on `origItemHeight.` To my knowledge this is the cleanest way to do this inline, although I'm open to being wrong

Reply:
TBH I've never seen that syntax before.
I'll merge this as it is. Thanks